samples/ts/21.react-media-template/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
New line at end. Can you run `npm run doctor`?

Reply:
`npm run doctor` doesn't seem to fix this